code/modules/clothing/outfits/ert.dm

```diff
@@ -460,3 +460,47 @@
 		/obj/item/gun/ballistic/automatic/sniper_rifle=1,\
 		/obj/item/gun/grenadelauncher/security=1,\
 		/obj/item/gun/ballistic/automatic/ar=1)
+
+/datum/outfit/hitman
+	name = "Hitman"
+
+	uniform = /obj/item/clothing/under/suit/black
+	shoes = /obj/item/clothing/shoes/sneakers/black
+	gloves = /obj/item/clothing/gloves/color/black
+	ears = /obj/item/radio/headset
+	glasses = /obj/item/clothing/glasses/sunglasses/advanced
+	l_hand = /obj/item/storage/secure/briefcase
+	id = /obj/item/card/id/syndicate
```

Comment:
i forget, is this the path for a true agent ID? do they have the proper creds to use it?

Reply:
This is the path for the true ID. I think since ERT are technically antags they might have access?